src/oge/data_pipeline.py

```diff
@@ -78,6 +79,8 @@ def main(args):
     args = get_args()
     year = args.year
 
+    os.environ["OGE_DATA_YEAR"] = "2022"
```

Comment:
I don't think we want to manually specify the year here - if anything, this should always be derived from `args.year`

Reply:
The `create_plant_ba_table`  function (now in the `oge.helpers` module) takes `year` as argument. Should it be set to 2022. I was trying to use the year that is used in the pipeline and that I cannot access in the `oge.helpers` module